tests/test_inclined_exponential.py

```diff
@@ -0,0 +1,102 @@
+# Copyright (c) 2012-2016 by the GalSim developers team on GitHub
+# https://github.com/GalSim-developers
+#
+# This file is part of GalSim: The modular galaxy image simulation toolkit.
+# https://github.com/GalSim-developers/GalSim
+#
+# GalSim is free software: redistribution and use in source and binary forms,
+# with or without modification, are permitted provided that the following
+# conditions are met:
+#
+# 1. Redistributions of source code must retain the above copyright notice, this
+#    list of conditions, and the disclaimer given in the accompanying LICENSE
+#    file.
+# 2. Redistributions in binary form must reproduce the above copyright notice,
+#    this list of conditions, and the disclaimer given in the documentation
+#    and/or other materials provided with the distribution.
+#
+
+from __future__ import print_function
+import numpy as np
+import os
+import sys
+
+from galsim_test_helpers import *
+
+try:
+    import galsim
+except ImportError:
+    path, filename = os.path.split(__file__)
+    sys.path.append(os.path.abspath(os.path.join(path, "..")))
+    import galsim
+
+# set up any necessary info for tests
+# Note that changes here should match changes to test image files
+image_dir = './inclined_exponential_images'
+
+# Values here are strings, so the filenames will be sure to work (without truncating zeros)
+image_inc_angles = ("0.0", "1.3", "0.2", "0.01", "0.1", "0.78")
+image_scale_radii = ("3.0", "3.0", "3.0", "3.0", "2.0", "2.0")
+image_scale_heights = ("0.3", "0.5", "0.5", "0.5", "1.0", "0.5")
+image_pos_angles = ("0.0", "0.0", "0.0", "0.0", "-0.2", "-0.2")
+image_nx = 64
+image_ny = 64
+oversampling = 1.0
+
+@timer
+def test_inclined_exponential():
+    """Test that the inclined exponential profile matches the results from Lance Miller's code."""
+    
+    for inc_angle, scale_radius, scale_height, pos_angle in zip(image_inc_angles,image_scale_radii,
+                                                                image_scale_heights,image_pos_angles):
+        image_filename = "galaxy_"+inc_angle+"_"+scale_radius+"_"+scale_height+"_"+pos_angle+".fits"
+        image = galsim.fits.read(image_filename, image_dir)
```

Comment:
I see that in the time since Mike made this comment, you included the code and README in the subdirectory.  I would still recommend adding a note here in ```test_inclined_exponential.py``` mentioning that the tests are against reference code provided by Lance Miller, and it can be found in the subdirectory with the test images.